plugins/circleci/src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Header, Page, pageTheme, HeaderLabel } from '@backstage/core';
+// @ts-ignore
+import logo from '../../assets/circle-logo-badge-white-15.png';
+import { Box } from '@material-ui/core';
+
+export const Layout: React.FC = ({ children }) => (
+  <Page theme={pageTheme.tool}>
+    <Header
+      pageTitleOverride="Circle CI Plugin"
+      title={
+        <Box display="flex" alignItems="center">
+          <img src={logo} style={{ height: '1em' }} />
+          <Box mr={1} /> Circle CI Plugin
```

Comment:
Remove "Plugin" form these titles. Plugins are not really a concept that users should need to know about.

Reply:
[x] removed